lib/MetricsConsumer.js

```diff
@@ -141,8 +188,10 @@ class MetricsConsumer {
         return done();
     }
 
-    _sendRequest(key, value) {
-        this._statsClient.reportNewRequest(key, value);
+    _sendRequest(action, site, redisKeys, keyType, value) {
+        if (redisKeys[keyType] && value && value !== 0) {
```

Comment:
last test `value !== 0` looks useless - actually could we want to publish even if `value === 0`?

Reply:
This sends a request to redis to save the given value. Since it won't change the value in redis, I thought it'd be nice to avoid sending altogether